docs/tex/gateway_kernel_init_sequence_diagram/Makefile

```diff
@@ -0,0 +1,13 @@
+fname=gateway_kernel_init_sequence_diagram
```

Comment:
Yes, but I figured that was kind of brittle since the odds of a valid texlive (with all the needed packages) existing by default in the doc build env has to be low.

I did just find `tectonic`. It apparently could be installed as part of the doc build conda env, is a self contained texlive, and pulls needed packages automagically. Seems a bit like overkill for one .png, but it'd probably work

Reply:
Let's just be sure to leave build instructions somewhere (perhaps commented out in docs/Makefile?).